front/app/components/ProjectAndFolderCards/index.tsx

```diff
@@ -30,10 +29,14 @@ const ProjectAndFolderCards = ({
   showSearch = false,
   ...otherProps
 }: Props) => {
+  // used locally to keep track of the depth of the search
+  const [search, setSearch] = useState<string | null>(null);
+  const rootLevelOnly = search && search.length > 0 ? false : true;
```

Comment:
Could you add a comment here too?

> When there's no search term, we only want to show root level admin publications (folders and projects that are not in a folder). When there is a search term we want to show all projects, including those inside folders.

Also, what do you think about`!search || search.length === 0` instead of `search && search.length > 0 ? false : true`?

Reply:
Added the comment and yeah that boolean seems fine, updated it